wallet-api/wallet-api.cabal

```diff
@@ -46,29 +63,42 @@ library
                  -Wredundant-constraints
     build-depends:
         base >=4.9 && <5,
-        ghc -any,
```

Comment:
Please read `CONTRIBUTING.md` about what to do when adding dependencies. You need to regenerate `pkgs` in this case.

Reply:
yes, currently running generate script, will no doubt take a few days